### ISSUE TITLE
completed-docs not working as expected with interfaces only

### ISSUE THREAD

**Issue by rodrijuarez** [palantir/tslint#4627]
### Bug Report

- __TSLint version__: 5.14.0
- __TypeScript version__: 3.1.6
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
// code snippet

/**
* Documentation
*/
interface ISomething {
  /**
  * Documentation
  */
  mySecondAttribute: string;
}

class Something {
  public myAttribute: string;

}

```

with `tslint.json` configuration:

```json
{
    "defaultSeverity": "error",
    "extends": [
        "tslint:recommended",
        "tslint-config-prettier"
    ],
    "jsRules": {},
    "rules": {
        "completed-docs": [true, "interfaces", "properties"],
        "prettier": [true, ".prettierrc"],
        "no-magic-numbers": [true, 0, 1, 2]
    },
    "rulesDirectory": [
        "tslint-plugin-prettier"
    ]
}

```

#### Actual behavior

```
Documentation must exist for properties.
```

#### Expected behavior

 No TSLint errors. I just want documentation for the properties of the interfaces and the interface itself

**Comment by ninahaack** [comment]
I'm having the same issue. I want to require documentation for the interface properties and I can't do it. The maximum that I get it's requirement for all properties using: `"completed-docs": [true, "interfaces", "properties"],`

**Comment by JoshuaKGoldberg** [comment]
Interesting. This configuration should work to only enforce documentation on interface properties:

```json
{
    "rules": {
        "completed-docs": [
            true,
            {
                "interfaces": {
                    "properties": true
                }
            }
        ]
    }
}
```

Does that work for you?

**Comment by ninahaack** [comment]
@JoshuaKGoldberg Hi, thank you for the response, but for me I still not complaining for the interfaces properties, only for the interface itself. So in this example it's fine:
```
/**
 * Foo
 */
export interface IFoo {
  foo: string;
}
```

but I want this as the correct one:
```
export interface IFoo {
 /**
  * Foo
  */
  foo: string;
}
```

**Comment by JoshuaKGoldberg** [comment]
☠️ *TSLint's time has come!* ☠️ 

TSLint is no longer accepting most feature requests per #4534. See **[typescript-eslint.io](https://typescript-eslint.io)** for the new, shiny way to lint your TypeScript code with **ESLint**. ✨ 

It was a pleasure open sourcing with you all!

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈 _(#4534)_  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
